contracts/Remittance.sol

```diff
@@ -0,0 +1,94 @@
+pragma solidity 0.5.10;
+
+import "./SafeMath.sol";
+import "./Stoppable.sol";
+
+contract Remittance is Stoppable {
+    using SafeMath for uint;
+    
+    struct Order {
+        uint deadline;
+        address emitter;
+        uint amount;
+    }
+    
+    uint constant TX_FEES = 2000;
+
+    mapping(bytes32 => Order) public orders;
+    mapping(address => uint) public fees;
+    
+    event LogNewOrderPlaced(address indexed emitter, uint amount, bytes32 hashedOTP);
+    event LogFeesTaken(address indexed emitter, address owner, uint amount);
+    event LogOrderCancelled(address indexed emitter, uint amount, bytes32 hashedOTP);
+    event LogWithdrawn(address indexed emitter, bytes32 hashedOTP, uint amount);
+    event LogFeesWithdrawn(address indexed emitter, uint amount);
+
+    function hashOTP(address exchange, bytes32 password) public view returns(bytes32 hash) {
+        require(exchange != address(0) && password != 0, "Both exchange and password must be set");
+        hash = keccak256(abi.encodePacked(address(this), exchange, password));
+    }
+    
+    function newOrder(bytes32 hashedOTP, uint delay) _onlyIfRunning public payable returns(bool success) {
+        require(hashedOTP != 0 && msg.value > 0, "An error occured. Ensure that the secret is set and that your transaction value is above 0");
+        require(delay > 0 days, "Delay should be above 0 day");
```

Comment:
I think `> 0` is the same, cheaper.
Also, on line 44, it is used as seconds. Which is good. You don't want conversions to be handled in the EVM.